src/models/mods.rs

```diff
@@ -73,13 +73,13 @@ pub struct Version {
     pub files: Vec<VersionFile>,
     /// A list of mods that this version depends on.
     pub dependencies: Vec<ModId>,
```

Comment:
Great idea. Theoretically it could just link to VersionId since those should be unique throughout all versions

Reply:
The issue with that is that minor patches will be in new versions, so depending on a specific version of the mod prevents bug fixes from being applied to that mod.  That also cause issues because I assume most mods don't support having multiple versions of themselves loaded at once - we may end up with a dependency hell sort of situation unless we deal with this carefully.

Reply:
Agreed. We could have it support a "Major" version of the mod i.e. 1.x instead of 1.1.x. However not all mod authors will follow this rule. I think it's probably best for the modpack authors to handle this and for the modder to just specify a specific version that they know will work.